monitor/monitoring-dam-generator-core/src/main/java/eu/seaclouds/monitor/monitoringDamGenerator/dcScriptGenerators/DataCollectorDeploymentScriptGenerator.java

```diff
@@ -0,0 +1,23 @@
+package eu.seaclouds.monitor.monitoringDamGenerator.dcScriptGenerators;
+
+import eu.seaclouds.monitor.monitoringDamGenerator.adpParsing.Module;
+
+public interface DataCollectorDeploymentScriptGenerator {
```

Comment:
Not sure I get the idea of the DataCollectorDeployemntScriptGenerator. My understanding is that a DC is yet another TOSCA NodeType. Similarly to any other NodeType, the DC should be defined as i.e. "seaclouds.node.datacollector.X" and so the "MonitoringDamGenerator" should simply attach the string as the deployer will do the deployment.
Am I missing something?